### PR TITLE
add types for aws-param-store

### PR DESCRIPTION
**Pull request by jasonthomasgray**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/aws-param-store/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Type definitions for aws-param-store 2.1
+// Project: https://github.com/vandium-io/aws-param-store#readme
+// Definitions by: Jason Gray <https://github.com/jasonthomasgray>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
```
Comment (PranavSenthilnathan) [inline types/aws-param-store/index.d.ts]: Is this needed?

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

File: types/aws-param-store/index.d.ts (hunk 1 above)
Comment (jasonthomasgray) [reply types/aws-param-store/index.d.ts]: Without this I was getting a "cannot find name 'Buffer'" error.
I assume that the aws-sdk is using some node types.

Is there a more correct way for me to solve this issue?

**Comment by jasonthomasgray** [comment]
@PranavSenthilnathan I'm not sure if I've commented in a way that typescript-bot doesn't understand.
The `/// <reference types="node" />` was the only way I managed to find to get my tests passing.
I attempted a search through the existing types to see if any other were referencing aws-sdk types and had similar issues but was unable to find any.

If there is a better way to handle this I would be happy to update.

File: types/aws-param-store/index.d.ts (hunk 1 above)
Comment (PranavSenthilnathan) [reply types/aws-param-store/index.d.ts]: Hmm I think it's ok since aws-sdk needs a dependency on node to begin with.

**Review by PranavSenthilnathan — APPROVED** [review]

**Comment by PranavSenthilnathan** [comment]
Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**